index.md

```diff
@@ -1,15 +1,55 @@
+
+![Google Search query result](/assets/images/dcoverview1.png){: width="800"}
+
+## A standards-based knowledge graph, schema, and APIs
+
+Data Commons needs to be able to stitch together data from disparate data sets in different formats and encodings, in a wide range of domains, from time series about demographics and employment, to hurricanes, to protein structures. To do so, it models the world as a [knowledge graph](https://blog.google/products/search/introducing-knowledge-graph-things-not/) consisting of nodes, or entities, with properties (attributes) and relationships between them forming directed edges between the nodes. The data model is based on the [Schema.org](https://www.schema.org) framework, an open framework used by over 40M websites; Data Commons makes heavy use of [Schema.org](https://www.schema.org/docs/schemas.html) constructs and extends the model as required, introducing both general constructs (such as intervals) and values for common properties. 
+
+The Data Commons [Knowledge Graph browser](https://datacommons.org/browser/) allows you to peek into the structure of the graph, and the APIs allow you to directly query the parts of the graph (e.g. nodes, triples, etc.).
+
+Importantly, numeric time series data are first-class entities, with "(statistical) variable" being an entity that represents a metric definition, and "observation" being an entity that represents the value of a variable at a specific time. The [Statistical Variable Explorer](https://datacommons.org/tools/statvar) allows you to browse existing variables, and the [Visualization tools](https://datacommons.org/tools/visualization) provide aggregated views of this data over time, geography, or 2-dimensional space. The APIs also allow you to directly query observations. 
+
+The knowledge graph is also mapped to relational tables that allow for [SQL querying](https://docs.datacommons.org/bigquery/) (requiring a [Google Cloud BigQuery](https://cloud.google.com/bigquery) account) and easier joining to other datasets outside of Data Commons. 
+
+<!--To learn more about the data model and key concepts, see [Key concepts](). -->
+
+## An open-source project and website platform
+
+Data Commons is intended to be a community-based resource, where individuals and organizations can contribute data, code, documentation and educational materials. Source code, schemas, and documentation are publicly available at [https://github.com/datacommonsorg](https://github.com/datacommonsorg). 
```

Comment:
Can we drop "intended" and just say that it is a community-based resource? We do have external contributions...

Reply:
Yes.